scripts/ingests/2020Best_publications_ingest.py

```diff
@@ -0,0 +1,180 @@
+import sys
+sys.path.append('.')
+from utils import *
+from astrodbkit2.astrodb import create_database
+from astrodbkit2.astrodb import Database
+from astropy.table import Table
+from pathlib import Path
+from simple.schema import *
+import os
+
+save_db = True #modifies .db file but not the data files
+RECREATE_DB = True #recreates the .db file from the data files
+VERBOSE = False
+
+verboseprint = print if VERBOSE else lambda *a, **k: None
+
+db_file = 'SIMPLE.db'
+db_file_path = Path(db_file)
+db_connection_string = 'sqlite:///SIMPLE.db'  # SQLite browser
+
+if RECREATE_DB and db_file_path.exists():
+    os.remove(db_file) #removes the current .db file if one already exists
+
+if not db_file_path.exists():
+    create_database(db_connection_string) #creates empty database based on the simple schema
+    db = Database(db_connection_string) #connects to the empty database
+    db.load_database('data/') #loads the data from the data files into the database
+else:
+    db = Database(db_connection_string) #if database already exists, connects to .db file
+
+
+add_Arti_bibcode = db.Publications.update().where(db.Publications.c.name == 'Arti15').\
+	values(bibcode='2015ApJ...806..254A')
+db.engine.execute(add_Arti_bibcode)
+db.Publications.delete().where(db.Publications.c.name == 'Barm10').execute()
+db.Publications.delete().where(db.Publications.c.name == 'Bouy06').execute()
+update_Deac11 = db.Publications.update().where(db.Publications.c.name == 'Deac11').\
+	values(name='Deac11b')
+db.engine.execute(update_Deac11)
+update_Delo17 = db.Publications.update().where(db.Publications.c.name == 'Delo17').\
+	values(name='Delo17b')
+db.engine.execute(update_Delo17)
+update_Folk07 = db.Publications.update().where(db.Publications.c.name == 'Folk07').\
+	values(name='Folk07PhD')
+db.engine.execute(update_Folk07)
+update_Gate09 = db.Publications.update().where(db.Publications.c.name == 'Gate08').\
+	values(name='Gate09')
+db.engine.execute(update_Gate09)
+add_Gauz15_bibcode = db.Publications.update().where(db.Publications.c.name == 'Gauz15').\
+	values(bibcode='2015ApJ...804...96G')
+db.engine.execute(add_Gauz15_bibcode)
+add_Hink15_bibcode = db.Publications.update().where(db.Publications.c.name == 'Hink15').\
+	values(bibcode='2015ApJ...805L..10H')
+db.engine.execute(add_Hink15_bibcode)
+add_Irwi91_bibcode = db.Publications.update().where(db.Publications.c.name == 'Irwi91').\
+	values(bibcode='1991MNRAS.252P..61I')
+db.engine.execute(add_Irwi91_bibcode)
+update_Kirk97 = db.Publications.update().where(db.Publications.c.name == 'Kirk97b').\
+	values(name='Kirk97a')
+db.engine.execute(update_Kirk97)
+update_Law06 = db.Publications.update().where(db.Publications.c.name == 'Law_06').\
+	values(name='Law_06b')
+db.engine.execute(update_Law06)
+db.Publications.delete().where(db.Publications.c.name == 'Legg17').execute()
```

Comment:
This is the action which caused the problem. I don't think we should change the script at this point, but just a lesson learned.